### PR TITLE
Update the <related packages> section in README.md

### PR DESCRIPTION
**Pull request by amontoison**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,8 +91,8 @@ fvals = pmap(evalmodel, probs)
 
 - [NLPModels.jl](https://github.com/JuliaSmoothOptimizers/NLPModels.jl) provides an
   AbstractModel from which CUTEst.jl derives, and other models deriving from it,
-  such as MathProgNLPModel, which uses
-  [MathProgBase.jl](https://github.com/JuliaOpt/MathProgBase.jl),
+  such as MathOptNLPModel, which uses
+  [MathOptInterface.jl](https://github.com/jump-dev/MathOptInterface.jl),
   SimpleNLPModel, for user created functions, ADNLPModel with automatic
```
Comment (abelsiqueira) [inline README.md:96]: Please remove SimpleNLPModel because it doesn't exist anymore.

File: README.md [hunk 2]
```diff
@@ -91,17 +91,16 @@ fvals = pmap(evalmodel, probs)
 
 - [NLPModels.jl](https://github.com/JuliaSmoothOptimizers/NLPModels.jl) provides an
   AbstractModel from which CUTEst.jl derives, and other models deriving from it,
-  such as MathProgNLPModel, which uses
-  [MathProgBase.jl](https://github.com/JuliaOpt/MathProgBase.jl),
-  SimpleNLPModel, for user created functions, ADNLPModel with automatic
-  differentiation, and SlackModel, which creates an
+  such as MathOptNLPModel, which uses
+  [MathOptInterface.jl](https://github.com/jump-dev/MathOptInterface.jl),
```
Comment (dpo) [inline README.md:95]: Make MathOptNLPModel a separate item here. It deserves it. It will also make this sentence understandable.

Comment (abelsiqueira) [reply README.md:95]: True, NLPModelsJuMP is separated from NLPModels.jl now.

**Comment by abelsiqueira** [comment]
THanks

**Pull request merged**